crypto/include/internal/md32_common.h

```diff
@@ -93,6 +93,12 @@
 # error "HASH_BLOCK_DATA_ORDER must be defined!"
 #endif
 
+#ifdef __GNUC__
+typedef unsigned int uint_a1 __attribute((__aligned__(1)));
+#else
+typedef unsigned int uint_a1;
+#endif
```

Comment:
Same comment as in camellia.c module. Same options. Difference is that while camellia can be viewed as non-critical, this affects multiple algorithm, which can be viewed as critical, most notably sha256. However! Given assembler coverage for SHA256, one can as well say that b) option, i.e. omission of whole section, is appropriate. So even here my suggestion would be b).

Reply:
#6508. Though I had to revert a commit to make travis output more sensible. So one should really look at https://github.com/openssl/openssl/pull/6508/commits/4332f9f7c00b52123599b45005c0fa7150815f9d alone.